protos/sp_common.proto

```diff
@@ -54,9 +54,9 @@ message Metric {
 
 message TailRequest {
   TailRequestType type = 1;
-  string id = 2;
+  optional string id = 2; // Filled out by server
```

Comment:
If filled out by server, should it be a `_id` field?

Reply:
Done!